tests/acceptance/10_files/02_maintain/005.cf

```diff
@@ -47,6 +47,7 @@ body perms test_og
 
 body copy_from test_copy
 {
+      preserve => "true";
```

Comment:
- Similar to 004 testfiles expected mode is based on queried mode for mocked
    etc/group source file.
    - So the test expected preservation of the mode, but it was not specified.